src/meta/src/barrier/info.rs

```diff
@@ -22,13 +22,16 @@ use crate::barrier::Command;
 use crate::manager::{ActiveStreamingWorkerNodes, ActorInfos, InflightFragmentInfo, WorkerId};
 use crate::model::{ActorId, FragmentId};
 
+#[derive(Debug, Clone)]
```

Comment:
Can we add some docs for this? Is it for any new fragments associated with the barrier command?

And what is the `is_injectable` field for, when is it true/false?

Reply:
`is_injectable` means whether we need to inject the barrier directly to the actors of the fragment. The struct is just extracted from the `CommandFragmentChanges::NewFragment`, and no logic is changed in this PR.

These fragments are the most upstream fragments, such as source, now.